README.md

```diff
@@ -13,6 +13,7 @@ Ensure you use versions that work together!
 
 | `expo` | `@daily-co/react-native-webrtc` | `@config-plugins/react-native-webrtc` | `@daily-co/react-native-daily-js` | `@daily-co/config-plugin-rn-daily-js` |
 |--------|---------------------------------|---------------------------------------|-----------------------------------|---------------------------------------|
+| 50.x   | 118.0.3-daily.1                 | 8.0.0                                 | 0.59.0                            | 0.0.4                                 |
```

Comment:
We just need to wait for `@daily-co/react-native-daily-js` `0.59.0` to be published before merging this PR.